File: WebApi/Models/Brokers/Broker.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace WebApi.Models.Broker
{
    public class Broker
    {
        public int BrokerID { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string PhoneNumber { get; set; }
        public string OfficeAddress { get; set; }
        public string LicenseNumber { get; set; }
        [Column(TypeName = "decimal(18, 4)")]
        public decimal CommissionRate { get; set; }
    }
}

File: Angular/src/app/shared/Broker.ts
export class Broker {
  brokerID: number = 0;
  name: string = "";
  surname: string = "";
  phoneNumber: string = "";
  officeAddress: string = "";
  licenseNumber: string = "";
  commissionRate: number = 0;
  documents : Document[] =[];
}


export class Document {
  documentID: number = 0;
  contractorID: number = 0;
  documentName: string = "";
  filePath : string ="";
  uploadDate : Date = new Date();
}

